### PR TITLE
Post Word Doc with String

### PR DESCRIPTION
**Pull request by akanex115**
## Related PRs
- _None_

## Checklist (Developers)
- [x] `Related PRs` is correctly modified, if it is not `None`
- [x] One of the followings is handled
  - At least one or more issue are linked to this PR under `development`
  - [The issue template](https://github.com/ajktown/.github/blob/main/issue_template.md) is directly copied above the `Related PRs`
    - You can directly copy background with issue from the samed/different repository.
  - One of the related PRs is linked to the issue under `development`
- [x] Assignee is set
- [x] Labels are set
- [x] Title is checked
- [x] `yarn inspect` is run
- [x] `TODOs` of associated issue (or TODOs here, if present) are handled and checked
- [x] Final Operation Check is done

## Checklist (Reviewers)
- [ ] Check if there are any other missing TODOs that are not yet listed
- [ ] Every item on the checklist has been addressed accordingly

### CODE REVIEW

File: dev/diagrams/use-post-word-with-string-hook.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Use Post Word With String Hook
```
Comment (mlajkim) [inline dev/diagrams/use-post-word-with-string-hook.md:1]: `on Post Word With String`?

File: dev/diagrams/use-post-word-with-string-hook.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+@startuml
+
+title usePostWordWithStringHook()
+
+box "FE" #Lightblue
+  participant "Component" as fe
+  participant "Hook" as hook
+  participant "IsWritingMode" as useState
+  participant "IsLoading" as useState2
```
Comment (mlajkim) [inline dev/diagrams/use-post-word-with-string-hook.md:24]: こういうstateは前回は理解させるため作ったので、
これからは書かなくていいかも

**Review by mlajkim — COMMENTED** [review]
とある人がこれを読んだ時に、結局赤の線の最後でどこにいけばいいのかがわからない
繋げてあげないといけない

image]

あと、`participant "End User" as user`どこいっちゃった？

**Comment by mlajkim** [comment]
Sample:
image

File: dev/diagrams/on-post-word-with-string-hook.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# On Post Word With String Hook
+
+<!-- TOC -->
+
+- [Use Post Word With String Hook](#use-post-word-with-string-hook)
+  - [Overview](#overview)
```
Comment (mlajkim) [inline dev/diagrams/on-post-word-with-string-hook.md:6]:
```suggestion
- [On Post Word With String Hook](#on-post-word-with-string-hook)
  - [Overview](#overview)
```

File: dev/diagrams/on-post-word-with-string-hook.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+			  deactivate api
+				usePostWord -> recoil: set(wordIdsState,[postedWord.id,...wordIds])
+				usePostWord -> recoil: set(wordsFamily(postedWord.id),postedWord)
+				usePostWord -> recoil: set(semestersState,semesters.semesters)
+				hook <- usePostWord: Returns nothing
+			deactivate usePostWord
+			fe <- hook: Returns nothing
+		deactivate hook
+		user <- fe: Returns nothing
+	deactivate fe
```
Comment (mlajkim) [inline dev/diagrams/on-post-word-with-string-hook.md:60]: Tabが大きい気がするね
image

**Review by mlajkim — COMMENTED** [review]
first review

File: dev/diagrams/on-post-word-with-string-hook.md (hunk 4 above)
Comment (akanex115) [reply dev/diagrams/on-post-word-with-string-hook.md:60]: indent size 変更しました。

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**